tools/find_image.rb

```diff
@@ -30,17 +30,17 @@ def run
       c.action do |args, options|
         options.default \
           :build_type => "nightly-",
-          :url => "https://openshift-release.svc.ci.openshift.org/"
+          :url => "https://amd64.ocp.releases.ci.openshift.org/"
```

Comment:
`amd64` is new subdomain for release page?

Reply:
@kedark3 that's what the ART folks told me going forward

Reply:
Ack.